File: lua/refactoring/tests/refactor/119/cpp/simple_string/extract_var.expected.cpp
#include <algorithm>
#include <iostream>
#include <string>

struct Person {
  std::string firstName;
  std::string lastName;
};

std::string orderCalculation(Person person, std::string start, std::string end) {
  auto foo_bar = " [space] ";
  std::string greeting = start + foo_bar + end;

  std::cout << greeting << std::endl;

  return start + person.firstName + foo_bar + person.lastName + end;
}

File: lua/refactoring/tests/refactor/119/go/simple_string/extract_var.expected.go
package main

import (
    "fmt"
)

type Person struct {
    firstName string
    lastName  string
}

func orderCalculation(person Person, start string, end string) string {
    space := " [space] "
    greeting := start + space + end

    fmt.Println(greeting)

    return start + person.firstName + space + person.lastName + end
}
